backend/pipelines/bmd_scraper/silver/transform.py

```diff
@@ -0,0 +1,411 @@
+                logger.info(f"Loaded bronze metadata from {metadata_path}")
+        
+        # Create mapping for status fields normalization
+        self.status_mapping = {
+            # Danish status values to standardized values
+            "Godkendt": "approved",
+            "Udgået": "expired", 
+            "Tilladt": "permitted",
+            "Ikke godkendt": "not_approved",
+            # Add more mappings as needed
+        }
+    
+    def read_excel(self) -> pd.DataFrame:
+        """
+        Read the raw Excel file into a pandas DataFrame.
+        
+        Returns:
+            DataFrame containing the raw data
+        """
+        logger.info(f"Reading Excel file from {self.input_file}")
+        try:
+            conn = duckdb.connect(database=':memory:')
+
+            df = conn.execute(f"SELECT * FROM read_xlsx('{self.input_file}', range = 'A4:Z', stop_at_empty = true, header=true);").fetchdf()
+            # Log the size of the dataframe
+            logger.info(f"Read {len(df)} rows and {len(df.columns)} columns")
+            conn.close()
+            # Store column information in metadata
+            self.silver_metadata = {
+                "transform_timestamp": datetime.now().isoformat(),
+                "source_file": str(self.input_file),
+                "bronze_timestamp": self.timestamp,
+                "row_count": len(df),
+                "column_count": len(df.columns),
+                "columns": list(df.columns),
+            }
+            
+            return df
+        
+        except Exception as e:
+            conn.close()
+            logger.exception(f"Error reading Excel file: {e}")
+            raise
+    
+    def clean_column_names(self, df: pd.DataFrame) -> pd.DataFrame:
+        """
+        Standardize column names to lowercase with underscores.
+        
+        Args:
+            df: Raw DataFrame
+            
+        Returns:
+            DataFrame with standardized column names
+        """
+        # Map of original column names to standardized names
+        column_mapping = {}
+        
+        for col in df.columns:
+            # Convert to lowercase, replace spaces with underscores
+            clean_col = col.lower().strip().replace(' ', '_')
+            # Remove special characters
+            clean_col = ''.join(c if c.isalnum() or c == '_' else '_' for c in clean_col)
+            # Remove consecutive underscores
+            while '__' in clean_col:
+                clean_col = clean_col.replace('__', '_')
+            # Remove leading/trailing underscores
+            clean_col = clean_col.strip('_')
+            
+            column_mapping[col] = clean_col
+        
+        # Rename columns
+        df = df.rename(columns=column_mapping)
+        
+        # Log the column mapping
+        logger.info(f"Column mapping: {column_mapping}")
+        
+        return df
+    
+    def clean_data(self, df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
could you ensure here that columns with semicolumn separated lists (bruger_pesticid, bruger_biocid,  Mindre anvendelse-nr., Mindre anvendelse-godkendelsesindehaver and maybe others) are appropriately separated (so are easily queriable later on)?